lib/views/home_content/map/post_map.dart

```diff
@@ -15,31 +17,38 @@ class PostMap extends StatelessWidget {
   });
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
+    final positionNotifier = ref.watch(mapProvider.notifier);
```

Comment:
I suggest we rename this to something clearer, like mapNotifier.

Reply:
FIxed with 22ce5e3